ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java

```diff
@@ -10,9 +10,10 @@
 
 package org.openmrs.reference;
 
+import static org.junit.Assert.assertTrue;
```

Comment:
Refactored this import to suit the existing imports order. Check through the existing classes to see how imports are organised